extensions/liquibase/runtime/src/main/java/io/quarkus/liquibase/runtime/graal/SubstituteStringUtil.java

```diff
@@ -13,4 +14,14 @@
     @RecomputeFieldValue(kind = RecomputeFieldValue.Kind.Reset)
```

Comment:
I wonder if we could use `NewInstance` instead of `Reset` and get rid of the method substitution.

Reply:
I will try it

Reply:
I remove the method and switch to `NewInstance` and got the following error:
```
Caused by: com.oracle.svm.core.jdk.UnsupportedFeatureError: Code that was considered unreachable by closed-world analysis was reached. The call to java.util.Random.nextInt(int) is not reachable.
        at com.oracle.svm.core.util.VMError.unsupportedFeature(VMError.java:86)
        at com.oracle.svm.core.snippets.SnippetRuntime.unsupportedFeature(SnippetRuntime.java:153)
        at liquibase.util.StringUtil.randomIdentifer(StringUtil.java:498)
        at liquibase.Scope.enter(Scope.java:191)
```

Reply:
@gsmet do you have any suggestion how to get rid of the method?

Reply:
@galderz any hint?

Apparently, if we use `RESET` we get a NPE and we have to reimplement the method using a local `SecureRandom` which is not very efficient. This looks a bit odd to me, I would have expected `RESET` to initialize the field properly.

Reply:
@galderz any chance you could have a look at this? Or have someone with native image knowledge to have a look? Thanks!